### DISCUSSION TITLE
Understanding implementation of local client

### DISCUSSION THREAD

**Discussion by siddhsql** [qdrant/qdrant-client#582] [General]
Hi,

I am trying to understand how the [local client](https://github.com/qdrant/qdrant-client/blob/master/qdrant_client/local/qdrant_local.py) works. Is this doing a brute force O(N) search instead of building an index? Where is the indexing code if it builds an index?

**Comment by joein** [comment]
Hi

Yes, it does brute force, it has numpy under the hood

**Comment by siddhsql** [comment]
thanks for letting me know.
